vcx/libvcx/src/disclosed_proof.rs

```diff
@@ -269,7 +270,7 @@ impl DisclosedProof {
 
     fn retrieve_credentials(&self) -> VcxResult<String> {
         trace!("DisclosedProof::set_state >>>");
-        if settings::test_indy_mode_enabled() { return Ok(CREDS_FROM_PROOF_REQ.to_string()); }
+        if settings::mock_indy_test_mode_enabled() { return Ok(CREDS_FROM_PROOF_REQ.to_string()); }
```

Comment:
I suggest to rename it once again.
`mock` always about test -- so I suggest to omit `test`.
Moreover, `mode` is unnecessary -- from `enabled` we can already consider that this is some mode that can be enabled or disabled.
So I suggest `indy_mocks_enabled`